pkg/handler/handler.go

```diff
@@ -178,6 +184,14 @@ func (o *OpenAPIService) RegisterOpenAPIVersionedService(servePath string, handl
 							return
 						}
 					}
+					// Set Content-Type header in the reponse
+					if accepts.SubType == subTypeProtobufDeprecated {
```

Comment:
rather than doing this `if` check and constructing the return header every time, would it make more sense to add `ReturnedContentType string` to the `accepted` items above, and do an unconditional `w.Header().Set("Content-Type", accepts.ReturnedContentType)` here?